dali/operators/numba_function/numba_func.cu

```diff
@@ -112,9 +112,41 @@ bool NumbaFuncImpl<GPUBackend>::SetupImpl(std::vector<OutputDesc> &output_desc,
   return true;
 }
 
+
+std::vector<void*> prepare_args(const dali::TensorList<dali::GPUBackend> &in, uint64_t *ptr, uint64_t shape_ptr) {
+  void** meminfo = new void*;
```

Comment:
Maybe you can have `vector<void*>` and `vector<ssize_t>`you would use to to manage the memory instead of using new/delete, and defining it in RunImpl.
Something like:
```
vector<void**> tmp_pptrs;
void<ssize_t> tmp_sizes;
tmp_ptrs.resize(tmp_ptrs.size() + 2);

args.push_back((void*)tmp_pptrs[tmp_ptrs.size()-2]);
args.push_back((void*)tmp_pptrs[tmp_ptrs.size()-1]);
```
Or something like that. I just don't feel that the raw new/delete is a good idea here.

Reply:
Done